### DISCUSSION TITLE
Confused about Nova and Whisper tiers in JS SDK

### DISCUSSION THREAD

**Discussion by drukerez** [deepgram/community#120] [General help] [answered]
Very excited but also confused about the new models.

1. When calling the API with `model: "nova"` I see in the response that the model is `{ name: 'general-nova', version: '2023-03-13.31000', arch: 'nova' }`. Is this the only mode/tier for nova or is there a large/enhanced one as well? What params do I use to call those?

2. When calling the API with `model: "whisper-large"` I see in the response that the model is `{ name: 'large-whisper', version: '2022-12-08.0', arch: 'whisper' }`. Does this refer to whisper-large-v2 or not? If not then how do I call large v2?

Thanks!

**Comment by SandraRodgers** [comment, marked as answer]
Hi @drukerez 

When using `model:nova`, you should not use `tier`. That's because the tier is nova. Nova is like an even more enhanced tier than enhanced.

For that reason, you could use a different Deeggram model with nova, such as the phonecall model. Here is a list of which models can use nova, in addition to their other tiers:

general - tiers=nova, enhanced, base
phonecall - tiers= nova, enhanced, base

As for whisper, that is very different because it's our hosted Whisper model that comes from OpenAI. There are no tiers with whisper, just sizes.

When you call `whisper-large`, that calls the model whisper-large-v2. You can see all the sizes and models they call here https://developers.deepgram.com/documentation/guides/deepgram-whisper-cloud/#enable-whisper-model-and-sizes
